src/tools/pkcs15-init.c

```diff
@@ -218,6 +219,7 @@ const struct option	options[] = {
 	{ "wait",		no_argument, NULL,		'w' },
 	{ "help",		no_argument, NULL,		'h' },
 	{ "verbose",		no_argument, NULL,		'v' },
+	{ "user-consent",	required_argument, NULL, OPT_USER_CONSENT},
```

Comment:
please rename this to something like *always-authenticate* and move it near `OPT_PRIVATE` and `OPT_SENSITIVE`. Don't forget to update `doc/tools/pkcs11-tool.1.xml`

Reply:
I'll move the argument close to similar arguments and update pkcs11-tool.1.xml. Wasn't aware of this file, maybe that caused Travics CI build to fail?

Let me explain why I chose to use name "user-consent" in pkcs15-init.
This a PKCS#15 tool, so I though it would be good to use PKCS#15 terminology. User might be looking at PKCS#15/ISO7816-15 instead of PKCS#11 when using this tool. The meaning of this parameter is a also a bit different in pkcs#15: 

"The userConsent field gives, in the case of a private object (or an object for which access conditions has been specified), the number of times an application may access the object without explicit consent from the user (e.g. a value of 3 indicates that a new authentication will be required before the first, the 4th, the 7th, etc. access). A value of 1 means that a new authentication is required before each access.". 

It is an integer value and a nonzero value doesn't mean "always authenticate", unless it is 1. With pkcs15init you can for example set --user-consent 5, which means re-authentication is required every fifth usage. I'm don't know which cards support it this way, that's just what the spec says.

On the other hand, it would be good to use unified terms between OpenSC's tools when possible, this would support naming it as you suggest. I just kind of found more reasons to use the pkcs15 name in this tool. 

Please confirm, considering my explanation, are you sure you want it to be renamed?

Reply:
Oh, I'm terribly sorry. I was stuck believing that this change was done to pkcs11-tool (where you're correctly using `always-auth`). In pkcs15-init, you're perfectly right that using `user-consent` is more consistent with the PKCS#15 interpretation.

To sum up, please leave your implementation as is. You just need to
- extend the xml files, from which the man pages are generated
- use the vendor specific implementation regarding PKCS#11 as suggested by Doug

Reply:
No problem and thank you for taking the time to review this.

I'll make the needed modifications.